src/components/Book.js

```diff
@@ -1,10 +1,16 @@
 import PropTypes from 'prop-types';
 
-const Book = ({ book }) => (
+// eslint-disable-next-line react/prop-types
+const Book = ({ book, handleRemoveBook }) => (
```

Comment:
- [ ] <b>REQUIRED:</b> You are not allowed to use `eslint-disable` unless it totally obvious that you have to. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix this error instead of ignoring it.